bazel-mode.el

```diff
@@ -83,9 +83,21 @@
      . 'font-lock-builtin-face)
     ;; Keywords for BUILD files are a subset of Starlark keywords. For details
     ;; see https://docs.bazel.build/versions/master/skylark/language.html.
-    (,(regexp-opt '("and" "else" "for" "if" "in" "load" "not" "or")
+    (,(regexp-opt '("and" "else" "for" "if" "in" "not" "or" "load"
```

Comment:
Where did you take this list from? I thought the lists at head were already exhaustive.

Reply:
From [here](https://github.com/bazelbuild/starlark/blob/master/spec.md#lexical-elements)

Reply:
ping @phst

Reply:
can you add a reference to that and mention that BUILD files have the same keywords?

Reply:
Ok